src/openlifu/db/database.py

```diff
@@ -300,6 +303,18 @@ def get_session_ids(self, subject_id):
             self.logger.warning("Sessions file not found for subject %s.", subject_id)
             return []
 
+    def get_volume_ids(self, subject_id):
```

Comment:
Better to unit test this since we are adding it here in this pr. Similarly for `write_volume_ids`